### PR TITLE
Net

### PR DESCRIPTION
**Pull request by sixtyfourktec**

### CODE REVIEW

**Comment by jkloetzke** [comment]
Looks good. I'd just like to move pure libraries to the `libs` folder:

recipes/net/libmnl.yaml -> recipes/libs/libmnl.yaml
recipes/net/libnftnl.yaml -> recipes/libs/libnftnl.yaml

**Comment by sixtyfourktec** [comment]
> Looks good. I'd just like to move pure libraries to the `libs` folder:
> 
> recipes/net/libmnl.yaml -> recipes/libs/libmnl.yaml recipes/net/libnftnl.yaml -> recipes/libs/libnftnl.yaml

Done

File: recipes/net/ntp.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        --without-ntpsnmpd \
+        --disable-ATOM \
+        --disable-SHM \
+        --with-sntp \
+        --with-crypto \
+        --enable-openssl-random \
+        --enable-linuxcaps
+
+    mkdir -p install/.bob
+    cat >install/.bob/ntpd.user-table <<EOF
```
Comment (jkloetzke) [inline recipes/net/ntp.yaml:40]: I would like to merge the infrastructure to handle these special files first. Just to be sure that it's the right approach and the handling stays that way...

File: recipes/net/iptables.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+inherit: [autotools]
+
+metaEnvironment:
+    PKG_VERSION: "1.8.9"
```
Comment (rhubert) [inline recipes/net/iptables.yaml:4]: could you please bump this tho 1.8.10?

Comment (sixtyfourktec) [reply recipes/net/iptables.yaml]: Yes, done.

**Comment by jkloetzke** [comment]
Thanks.

**Pull request merged**